### PR TITLE
Remove deprecations that were not flagged in the 3.0 beta

### PR DESCRIPTION
**Pull request by katbyte**

### CODE REVIEW

File: internal/services/network/vpn_server_configuration_resource.go [hunk 1]
```diff
@@ -297,71 +297,6 @@ func resourceVPNServerConfiguration() *pluginsdk.Resource {
 						},
 					},
 				},
-				ConflictsWith: []string{
-					"radius_server",
-				},
-			},
-
-			"radius_server": {
```
Comment (mbfrahry) [inline internal/services/network/vpn_server_configuration_resource.go:305]: We can remove this from the docs

File: internal/services/redisenterprise/redis_enterprise_cluster_resource.go [hunk 2]
```diff
@@ -99,6 +99,7 @@ func resourceRedisEnterpriseCluster() *pluginsdk.Resource {
 
 			// RP currently does not return this value, but will in the near future
 			// https://github.com/Azure/azure-sdk-for-go/issues/14420
+			// TODO 3.0 remove or keep?
```
Comment (manicminer) [inline internal/services/redisenterprise/redis_enterprise_cluster_resource.go:102]: Can probably remove it at this point?

**Review by manicminer — APPROVED** [review]
Awesome, LGTM! 🥄

**Review by mbfrahry — APPROVED** [review]
LGTM!

**Pull request merged**